ord.go

```diff
@@ -0,0 +1,39 @@
+package gophplib
+
+import (
+	"fmt"
+	"reflect"
+)
+
+// Ord is a ported functions that works exactly the same as PHP 5.6's ord function.
+// In PHP 5.6, when the ord() function is used with a data type other
+// than a string, it automatically converts the given variable into a string
+// before processing it. To achieve the same behavior in Go,
+// this function converts an argument to string using the ZendParseArg() function.
+// For more information, see the [official PHP documentation].
+//
+// Reference :
+//   - https://github.com/php/php-src/blob/php-5.6.40/ext/standard/string.c#L2666-L2676
+//
+// Test Cases:
+//   - https://github.com/php/php-src/blob/php-5.6.40/ext/standard/tests/strings/ord_basic.phpt
+//   - https://github.com/php/php-src/blob/php-5.6.40/ext/standard/tests/strings/ord_error.phpt
+//   - https://github.com/php/php-src/blob/php-5.6.40/ext/standard/tests/strings/ord_variation1.phpt
+//
+// [official PHP documentation]: https://www.php.net/manual/en/function.ord.php
+func Ord(character any) (any, error) {
```

Comment:
`PHP_FUNCTION(ord)` 함수의 리턴을 보면 `RETURN_LONG`인데, 여기서는 리턴이 `any`인 이유가 있을까요?

Reply:
Original `Ord` function used to return either an int type or nil, which is why its return type was defined as any. However, I decided to change it to be defined as *int because it appears to be a safer

Reply:
I thought `uint8` would be clearer than `int` because the range of `uint8` is the same as the return value of the PHP `ord` function, and developers can easily cast it to `int` or `int32` or whatever, depending on where they're using it. So I thought it would be better to just return it as `uint8` and let them do the conversion as needed.

Reply:
I prefer `byte` over `uint8`